app/src/main/java/de/danoeh/antennapod/fragment/CoverFragment.java

```diff
@@ -63,12 +63,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     }
 
     private void loadMediaInfo() {
-        if(imgvCover == null) {
```

Comment:
Well, we've seen people with custom ROMs that don't always follow that lifecycle exactly. It just seemed like we could have left that check there just in case (I doubt this is a performance sensitive bit of code).

But we'll go ahead and see.